0000-template.md

```diff
@@ -48,9 +76,9 @@ on the impact of the API churn on existing apps, etc.
 
 > What other designs have been considered? What is the impact of not doing this?
 
-> This section could also include prior art, that is, how other frameworks in the same domain have solved this problem.
+> This section could also include prior art, that is, how other frameworks in the
+> same domain have solved this problem differently.
 
 ## Unresolved questions
 
-> Optional, but suggested for first drafts. What parts of the design are still
-TBD?
\ No newline at end of file
+> Optional, but suggested for first drafts. What parts of the design are still TBD?
```

Comment:
I think this is where we need the most prodding, since questions are what reviewers will put forth. AKA, current workarounds (if any) and how they're not enough/acceptable.

Reply:
> current workarounds (if any) and how they're not enough/acceptable

I thought of this as being something to go in the motivation section. I had "What does it look like to accomplish some set of goals today and how could that be improved?", but could explicitly call out workarounds if you think that'd help clarify

Reply:
Ah yea, sorry. Maybe being more direct with "What are the current workarounds, if any" is a better prompt/trigger-word?

Reply:
cool. I just added that